src/test/java/strategy/impl/BalanceOperationImplTest.java

```diff
@@ -0,0 +1,45 @@
+package strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import db.Storage;
+import model.FruitTransaction;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationImplTest {
+    private static final String BANANA = "banana";
+    private static final int OPERATION_AMOUNT = 10;
+    private static FruitTransaction fruitTransactionOk;
+    private static FruitTransaction fruitTransactionNotOk;
+
+    @BeforeClass
+    public static void beforeAll() {
+        fruitTransactionOk = new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                BANANA, OPERATION_AMOUNT);
+        fruitTransactionNotOk = new FruitTransaction(FruitTransaction.Operation.PURCHASE,
+                BANANA, OPERATION_AMOUNT);
+    }
+
+    @Test
+    public void handle_balance_ok() {
+        new BalanceOperationImpl().handler(fruitTransactionOk);

```

Comment:
Not fixed. You should have BalanceOperationImpl as a class-level variable and just set it up in the beforeAll method and then just reuse it. That's it. Fix it in all places